documentation/player-portal/android/player-config-android.md

```diff
@@ -35,14 +50,25 @@ Once you have a PKMediaEntry ready, you can build the player configuration and p
 
 ### About the PKMediaEntry  
 
-The PkMediaEntry holds information gathered from the media details and needed for the player, such as the URL to play, the DRM data, and duration.
+#### NOTE:
+
+* Building the `PKMediaEntry`  manullay is more relevant for customers which do not use Kaltura BE to store their content.
+ 
+* Once content is stored in Kaltura's BE you can use our `Media Providers` to fetch the PKMediaEntry 
+
+The PkMediaEntry holds information gathered from the media API's and needed for the player and plugins to function properly, such as the URL to play, the DRM data, duration and the type of the media.
```

Comment:
Not sure what you mean by "the media API's"...

Reply:
fixed